### ISSUE TITLE
Seeing this warning: Redis#sadd will always return an Integer in Redis 5.0.0. Use Redis#sadd? instead

### ISSUE THREAD

**Issue by kapso** [mperham/sidekiq#5493]
Ruby version: 3.1.2
Rails version: 7.0.3.1
Sidekiq / Pro / Enterprise version(s): Sidekiq v6.5.5 / Sidekiq Pro v5.5.2

```
Redis#sadd will always return an Integer in Redis 5.0.0. Use Redis#sadd? instead.(called from: /app/vendor/bundle/ruby/3.1.0/gems/sidekiq-6.5.5/lib/sidekiq/client.rb:234:in `atomic_push')
```

**Comment by mperham** [comment]
Please look at closed issues before opening issues for deprecations in
other gems.

**Comment by bf4** [comment]
xref https://github.com/mperham/sidekiq/issues/5484

**Comment by kroehre** [comment]
@mperham I definitely get where you're coming from but in @kapso's defense, I only found #5484 via this issue (thanks @bf4) since it was the only result when searching for that particular deprecation message. Github's search could be better 😄

**Comment by Robfaldo** [comment]
I also only found #5484 through this one. 

I've not been able to upgrade sidekiq because of dependency issues so I'll share a temporary fix that works for me locally so that I can develop without all the noise from this warning that might help others. I just went into the redis gem locally and could see 2 of these:

```
          ::Redis.deprecate!(
            "Redis#sadd will always return an Integer in Redis 5.0.0. Use Redis#sadd? instead." \
            "(called from: #{caller(1, 1).first})"
          )
```

which I commented out. I can't see this being a problem or affecting any behaviour and it's meant that I can use pry in my sidekiq without everything I write disappearing every 5 seconds when this warning gets logged :D
